src/app/modules/core/manager/tenant/components/container/tenant-container.component.ts

```diff
@@ -1,12 +1,14 @@
 import { Component, OnInit } from '@angular/core'
 
+import { InMemoryTenantService } from './../../../../../shared/services/tenant/in-memory-tenant.service'
```

Comment:
Questo import andrebbe sistemato in questo modo '@modules/shared/services/tenant/in-memory-tenant.service'